pkg/metricutil/metricutil.go

```diff
@@ -42,6 +42,7 @@ func GetCmdLabel(request *pdpb.Request) string {
 	label, ok := cmdLabels[name]
 	if !ok {
 		label = convertName(name)
+		cmdLabels[name] = label
```

Comment:
Need Mutex if you want to update the map after init. I recommend keeping it as is, because it should be rare to receive unknown enum.